kernel/fs/fs.go

```diff
@@ -112,28 +110,31 @@ func (s *Service) Initialize() {
 
 	must(s.fsys.(*mountablefs.FS).Mount(memfs.New(), "/sys/tmp"))
 
+	// Mount githubfs if user has gh_token
 	u, err := jsutil.WanixSyscall("host.currentUser")
-	if err != nil || u.IsNull() {
-		return
-	}
-	m := u.Get("user_metadata")
-	if m.IsUndefined() {
-		return
-	}
-	token := m.Get("gh_token")
-	if token.IsUndefined() {
-		return
+	if err == nil && !u.IsNull() {
```

Comment:
nope. this is bad go and less than ideal code in general. 
https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html
https://google.github.io/styleguide/go/decisions#indent-error-flow